### PR TITLE
[#29] Relative box sizes for quantities

### PR DESCRIPTION
**Pull request by riyaGit**
This following is a sample of the implementation:
- For small quantities, a tooltip is used to show the quantity  
Screen Shot 2019-10-23 at 4 49 53 PM

### CODE REVIEW

File: app/src/components/TimestampOrderBookScroller.js [hunk 1]
```diff
@@ -59,7 +60,19 @@ class TimestampOrderBookScroller extends Component {
             });
         });
 
-        this.setState({listItems}, () => {
+        if (listItems.length > 0){
+            listItems.map(listItem =>{
```
Comment (Lercerss) [inline app/src/components/TimestampOrderBookScroller.js]: We can probably do this in a single pass in the `map`s on lines 55 & 47

Comment (riyaGit) [reply app/src/components/TimestampOrderBookScroller.js]: sure

File: app/src/components/TimestampOrderBookScroller.js [hunk 2]
```diff
@@ -83,7 +96,8 @@ class TimestampOrderBookScroller extends Component {
     };
 
     render() {
-        const {listItems} = this.state;
+        const {listItems, maxQuantitySum} = this.state;
+        const leeway = maxQuantitySum + maxQuantitySum*(35/100);
```
Comment (alvyn279) [inline app/src/components/TimestampOrderBookScroller.js]: please rename this, and extract 35 % as a constant to express leeway

File: app/src/components/Order.js [hunk 3]
```diff
@@ -3,16 +3,24 @@ import {withStyles} from '@material-ui/core/styles';
...
         return (
-            <span className={classNames(classes.rectangle, type === 'bid' ? classes.bid : classes.ask)}>
-                {quantity}
-            </span>
+            <Box className={classNames(classes.rectangle, type === 'bid' ? classes.bid : classes.ask)} style={{minWidth:`${quantityBoxSize}%`, maxWidth: `${quantityBoxSize}%`}}>
+                {quantityBoxSize > 1.5 ?
+                    (
+                        <Typography className={classes.text}>{quantity}</Typography>
+                    ) : (
+                        <Tooltip title={quantity} placement='cursor'><span style={{width: '100%', height: '100%'}}/></Tooltip>
```
Comment (alvyn279) [inline app/src/components/Order.js]: can you put the width and height the stylesheet ?, would probably need to use `className`

**Pull request merged**